### ISSUE TITLE
Pattern Submission: Sharp Angled Cover Block Overlay

### ISSUE THREAD

**Issue by cavalierlife** [WordPress/pattern-directory#222]
This pattern makes use of the cover block's background gradient overlay in a way that many users might not be familiar with. By setting the location of the two points to approximately 50% with one being nearly opaque, and one being completely transparent, a sharp angular dividing line can be created. This is a style that many beginners would like to achieve but don't know how.

---

## Pattern Title
Cover block with sharp angled background overlay

## Pattern Categories
Header, Images

## Screenshots
![cover-block-pattern-angled-bkg-overlay](https://user-images.githubusercontent.com/28687464/122679630-3c899c00-d1b1-11eb-91f7-a046d6d1a97a.jpeg)

## Image Credits
https://www.goodfreephotos.com/cuba/other-cuba/waves-crashing-over-the-rocks-in-nature-in-cuba.jpg.php

## Block Markup

<div class="wp-block-cover alignfull has-background-dim-100 has-background-dim has-parallax has-background-gradient" style="background-image:url(https://devementor.com/wp-content/uploads/2021/06/wavesrocks-scaled.jpeg);min-height:857px"><span aria-hidden="true" class="wp-block-cover__gradient-background" style="background:linear-gradient(134deg,rgba(0,0,0,0.79) 47%,rgba(155,82,224,0) 49%)"></span><div class="wp-block-cover__inner-container">
<div class="wp-block-columns alignfull">
<div class="wp-block-column" style="flex-basis:35%">
<h2 class="has-white-color has-text-color" style="font-size:44px">Life's journey sometimes crashes on the rocks</h2>

<div class="wp-block-buttons">
<div class="wp-block-button has-custom-width wp-block-button__width-75 has-custom-font-size is-style-fill has-normal-font-size"><a class="wp-block-button__link has-white-color has-text-color has-background" style="border-radius:20px;background-color:#1080b8">Discover the beauty of the rocks</a></div>
</div>
</div>

<div class="wp-block-column"></div>
</div>
</div></div>

**Comment by kjellr** [comment]
Thanks for the submission, @cavalierlife! I've made an attempt to revise this a bit with imagery and copy that fits more closely with the rest of the images in this pre-release collection of patterns. I also added a sharp edge to the gradient, for a more graphic effect: 

<img width="1123" alt="Screen Shot 2021-06-24 at 2 24 04 PM" src="https://user-images.githubusercontent.com/1202812/123314051-1e48d680-d4f8-11eb-9e18-6821b2df4ac9.png">

Placing the text in the upper left ensures that it works nicely on mobile too: 

<img width="217" alt="Screen Shot 2021-06-24 at 2 24 27 PM" src="https://user-images.githubusercontent.com/1202812/123314060-20ab3080-d4f8-11eb-8540-f9f3df4a0067.png">

Looking forward to feedback. 

---

Image: Colombes et lis, étoffe imprimée from L'animal dans la décoration (1897) illustrated by Maurice Pillard Verneuil. CC0.
https://www.rawpixel.com/image/2053817/illustration-from-lanimal-dans-decoration

Markup: https://gist.github.com/kjellr/3785cc104cd89e4631ab0dc3f6e7e2dd
